### ISSUE TITLE
[Metaeffect Suggestion] Parallel Percentages

### ISSUE THREAD

**Issue by SeanMirrsen** [gta-chaos-mod/ChaosModV#597]
Alternate names: Individual Probability

Duration: Long

When the metaeffect is active, all probabilities of effect activation are treated as individual trigger probabilities. What this means is that, if there are three effects to choose from (as it is without Twitch voting), then **each** effect has a 33% chance to activate. So at least one effect will almost certainly activate, but two or three of the possible effects can also activate if they hit their respective percentage, all activating in parallel.

Same with Twitch voting, except the different probabilities are applied according to current votes.

**Comment by kolyaventuri** [comment]
I'm a little lost... So we iterate through each possible option, A, B, C, and roll the dice on if it activates separately?

**Comment by SeanMirrsen** [comment]
Yes. So that the percentage chances are not indicative of the *relative* probability of which effect is chosen to activate, but rather the *absolute* probability of a given effect activating on that timer cycle.

So that if you have 100 votes and it's 50 for #1, 30 for #2, and 10 each for #3 and #4, you roll four numbers 1-100, and if the first one is under 50 the first option activates, if the second is under 30 the second option activates, etc. Rather than one roll 1-100 and a check whether it lands in 1-50, 51-80, 81-90 or 91-100 like for the normal proportional voting.
